test/unistd/access.c

```diff
@@ -65,14 +66,73 @@ int main() {
   printf("F_OK(%s): %d\n", "renamedfile", faccessat(AT_FDCWD, "renamedfile", F_OK, 0));
   printf("errno: %d\n", errno);
 
+  struct stat fileStats;
+  stat("fchmodtest", &fileStats);
+#if !NODE*FS
```

Comment:
I don't think this works as you expect - it doesn't match `NODEFS` and `NODERAWFS`. I think it does `!((NODE) * (FS))` that is, a multiply followed by a negation. But maybe I'm confused - I'm not sure why it doesn't error on not having `NODE` or `FS` defined.

What I'd expect would work is
```
#if !defined(NODE) && !defined(NODERAWFS)
```
That's the explicit way to check for multiple things being or not being defined in the C preprocessor.

Reply:
That's pretty interesting. Is there a reason we must use `defined(NODEFS)` here, but it was ok to use `#if WASMFS` for some of the other ifdefs I had?

Reply:
I think technically both are ok, but maybe it is clearer to use `defined()` when the question is whether the symbol is defined or not (as opposed to whether it is `0` or `1`, that is, always defined as one of those values).

So it's probably ok to do `if !NODEFS && !NODERAWFS`. But if that doesn't work, you might need `defined()`.